### ISSUE TITLE
Poor Reconstruction Results on Custom Data

### ISSUE THREAD

**Issue by destroy314** [hustvl/4DGaussians#168]
Thank you for sharing your work. I am attempting to reconstruct a desktop scene using a 6-camera setup as shown in the image below.
![output](https://github.com/user-attachments/assets/a8850f4d-e467-4662-b12c-38e0bd83801c)

Based on the configuration in `multipleview/default.py`, I made some modifications: using half the learning rate, 7k coarse iterations, and 50k iterations. As shown in the video, the moving objects appear somewhat blurry in the input views, and some view overfitting can be observed in the novel views.

https://github.com/user-attachments/assets/fd2a6d9d-6c32-4fe3-ba7d-edd7058065aa

https://github.com/user-attachments/assets/df0b0ee9-40d5-410b-b80b-12baebc48133

Do you have any suggestions or hypotheses regarding this issue? I would greatly appreciate your help.

**Comment by guanjunwu** [comment]
Interesting. In fact,  the experiment conducted on 4D-GS is easy and simple, this is a hard and complex/large motion. If you can set a **strict multiview**, I recommend:
1. Add more cameras.
2. try to use multiview-based methods. such as: 4K4D: Real-Time 4D View Synthesis at 4K Resolution; [3DGStream](https://github.com/SJoJoK/3DGStream)

**Comment by MikeAiJF** [comment]
> 我有一个视频，如何制作数据集？

你好，你是多视角视频吗

**Comment by MikeAiJF** [comment]
> 感谢您分享您的作品。我正在尝试使用 6 摄像头设置重建桌面场景，如下图所示。 ![输出](https://private-user-images.githubusercontent.com/35775292/349456857-a8850f4d-e467-4662-b12c-38e0bd83801c.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjk4NDgxODMsIm5iZiI6MTcyOTg0Nzg4MywicGF0aCI6Ii8zNTc3NTI5Mi8zNDk0NTY4NTctYTg4NTBmNGQtZTQ2Ny00NjYyLWIxMmMtMzhlMGJkODM4MDFjLmpwZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMjUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDI1VDA5MTgwM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTQ3ZmY3ZDBkZmZmY2RhNWJiZWFhOWZjNDFkZDZiMmY5ZDQwMGI4MGEzZDcyMWYzYzM0NzU1MDg5NjUxN2FmNWEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.aESRPxAGcS8gsApl9uZiK-jcUfgLa0E4g9vTNlw6hjw)
> 
> 基于 中的配置`multipleview/default.py`，我做了一些修改：使用一半的学习率、7k 次粗迭代和 50k 次迭代。如视频中所示，移动物体在输入视图中显得有些模糊，并且在新视图中可以观察到一些视图过度拟合。
> 
>  摄像头2.mp4 
>  视频.mp4 
> 您对这个问题有什么建议或假设吗？我将非常感谢您的帮助。

你好，请问你的视频怎么处理的，希望得到你的帮助

**Comment by destroy314** [comment]
@MikeAiJF 简单的用ffmpeg下采样成图片，没有校色、统一曝光和白平衡

**Comment by destroy314** [comment]
looks like zeroNVS is better in this setting ;) https://arxiv.org/abs/2409.03685

**Comment by MikeAiJF** [comment]
> @MikeAiJF简单的用ffmpeg下采样成图片，没有校色、统一曝光和白平衡
多目视频我已经切割成图片了，我不知道接下来怎么进行处理了。才能进行训练。谢谢！

**Comment by destroy314** [comment]
> > @MikeAiJF简单的用ffmpeg下采样成图片，没有校色、统一曝光和白平衡
> > 多目视频我已经切割成图片了，我不知道接下来怎么进行处理了。才能进行训练。谢谢！

你可以参考本仓库readme中的**For multipleviews scenes**部分，或者手动运行`multipleviewprogress.sh`中的colmap命令（我是这样做的，因为默认参数匹配很容易失败，好像是SiftExtraction.estimate_affine_shape导致的）

**Comment by MikeAiJF** [comment]
> > > @MikeAiJF简单的用ffmpeg下采样成图片，没有校色、统一曝光和白平衡
> > > 多目视频我已经切割成图片了，我不知道接下来怎么进行处理了。才能进行训练。谢谢！
> 
> 你可以参考本仓库自述文件中的**对于多个视图场景**部分，或者手动运行`multipleviewprogress.sh`中的colmap命令（我就是这么做的，因为默认参数匹配很容易失败，好像是SiftExtraction.estimate_affine_shape导致的）
我一键运行这个脚本，一直找不到poses_bounds_multipleview.npy这个文件

**Comment by Harryhaozhu** [comment]
> > > [@MikeAiJF](https://github.com/MikeAiJF)简单的用ffmpeg下采样成图片，没有校色、统一曝光和白平衡
> > > 多目视频我已经切割成图片了，我不知道接下来怎么进行处理了。才能进行训练。谢谢！
> 
> 你可以参考本仓库自述文件中的**对于多个视图场景**部分，或者手动运行`multipleviewprogress.sh`中的colmap命令（我就是这么做的，因为默认参数匹配很容易失败，好像是SiftExtraction.estimate_affine_shape导致的）

您好，请问您使用multipleviewprogress.sh生成的位姿准确吗，我生成的有问题，导致重建出来效果特别差
